packages/contracts-periphery/hardhat.config.ts

```diff
@@ -18,15 +20,57 @@ const config: HardhatUserConfig = {
   networks: {
     optimism: {
       chainId: 10,
-      url: 'https://mainnet.optimsim.io',
+      url: 'https://mainnet.optimism.io',
+      verify: {
+        etherscan: {
+          apiKey: getenv('OPTIMISTIC_ETHERSCAN_API_KEY'),
+        },
+      },
     },
     opkovan: {
       chainId: 69,
       url: 'https://kovan.optimism.io',
+      verify: {
+        etherscan: {
+          apiKey: getenv('OPTIMISTIC_ETHERSCAN_API_KEY'),
+        },
+      },
     },
-    mainnet: {
+    ethereum: {
       chainId: 1,
-      url: 'https://rpc.ankr.com/eth',
+      url: `https://mainnet.infura.io/v3/${getenv('INFURA_PROJECT_ID')}`,
+      verify: {
+        etherscan: {
+          apiKey: getenv('ETHEREUM_ETHERSCAN_API_KEY'),
+        },
+      },
+    },
+    goerli: {
+      chainId: 5,
+      url: `https://goerli.infura.io/v3/${getenv('INFURA_PROJECT_ID')}`,
+      verify: {
+        etherscan: {
+          apiKey: getenv('ETHEREUM_ETHERSCAN_API_KEY'),
+        },
+      },
+    },
+    ropsten: {
+      chainId: 3,
+      url: `https://ropsten.infura.io/v3/${getenv('INFURA_PROJECT_ID')}`,
```

Comment:
I don't love coupling this to infura but it is pretty simple. We have similar hardhat config across many packages, maybe we want to export a base config that does all this stuff from a package that takes an overrides object and export that from a package so we can use a nice config across all contracts related repos